sync-core/src/main/java/com/cloudant/android/Base64OutputStreamFactory.java

```diff
@@ -28,7 +28,7 @@ public static OutputStream get(OutputStream os) {
             if (Misc.isRunningOnAndroid()) {
                 Class c = Class.forName("android.util.Base64OutputStream");
                 Constructor ctor = c.getDeclaredConstructor(OutputStream.class, int.class);
-                return (OutputStream)ctor.newInstance(os, 0);
+                return (OutputStream)ctor.newInstance(os, 2);
```

Comment:
We can, but I really don't see why we should. Its a lot of code execution to get the value 2, and it's not going to improve readability. It's not going to change between SDKs since the Java Compiler should treat `Base64.NO_WRAP` as a compile time constant, since its defined as `static final int`